bin/fuel-convert

```diff
@@ -10,6 +10,18 @@ from fuel.converters.base import MissingInputFiles
 from fuel.datasets import H5PYDataset
 
 
+class SplitOutputAction(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        absolute_path = os.path.abspath(values)
+        if os.path.isdir(absolute_path):
```

Comment:
I'm not totally sure about this. The behaviour then depends on what's on the filesystem, which is kind of strange... see what @rizar or @bartvm think.